app/src/main/java/fe/linksheet/module/viewmodel/PreferredAppSettingsViewModel.kt

```diff
@@ -53,8 +52,7 @@ class PreferredAppSettingsViewModel(
     val appsExceptPreferred = preferredApps.map { apps ->
         val preferredAppsPackages = apps.mapToSet { it.first.packageName }
         domainVerificationManager!!.getDisplayActivityInfos(context) {
-            it.hasVerifiedDomains(domainVerificationManager!!, true)
-                    && it.activityInfo.packageName !in preferredAppsPackages
+            it.activityInfo.packageName !in preferredAppsPackages
```

Comment:
Change here is so that Mastodon and Lemmy Redirect will actually show up as selectable, since they don't use `autoVerify="true"` because of performance issues with so many domains.